test/run-docker.sh

```diff
@@ -34,18 +34,13 @@ if [[ "$(is_running boulder-mysql)" != "true" ]]; then
 	# bring up mysql mariadb container - no need to publish port
 	# 3306 with host networking
 	docker run -d \
-		--net host \
 		-e MYSQL_ALLOW_EMPTY_PASSWORD=yes \
 		--name boulder-mysql \
-		mariadb:10.0 mysqld --bind-address=127.0.0.1
+		mariadb:10.0 mysqld
```

Comment:
The patch should use here --bind-address=0.0.0.0. Otherwise mysqld listens on all IPv4 and IPv6 addresses so when docker runs with IPv6 enabled that assigns globally routable IPv6 addresses to the container the container will be exposed to the Internet again. This is due to unfortunate feature with Docker ipv6 implementation that does not block connections to unpublished ports from outside world.